styles.css

```diff
@@ -0,0 +1,105 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2vh 5vw;
+}
+
+.vector_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.welcome {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.vector {
+  fill: rgb(0, 0, 0);
+  stroke-width: 1;
+  stroke: rgb(0, 0, 0);
+  width: 40px;
+  height: 3px;
+}
+
+.vectSvg {
+  width: 40px;
+  height: 3px;
+}
```

Comment:
- [ ] Kindly the hamburger you have choose is bigger than the Figma template. as requested by the previous review make changes and let's see your project approved so far